lib/matplotlib/axes/_axes.py

```diff
@@ -5435,7 +5435,8 @@ def get_interp_point(ind):
     def imshow(self, X, cmap=None, norm=None, aspect=None,
                interpolation=None, alpha=None, vmin=None, vmax=None,
                origin=None, extent=None, shape=None, filternorm=1,
-               filterrad=4.0, imlim=None, resample=None, url=None, **kwargs):
+               filterrad=4.0, imlim=None, resample=None,
+               antialiased=True, url=None, **kwargs):
```

Comment:
This needs to go at the end (or be kwarg only) to not change the API.

Reply:
ooops.    Also, I should have made a tentative PR....

Reply:
Fixed...